ui/components/ui/icon-with-fallback/icon-with-fallback.component.js

```diff
@@ -2,6 +2,15 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+/**
+ * @deprecated The `<IconWithFallback />` component has been deprecated in favor of the new `<AvatarNetwork or AvatarFavicon>` component from the component-library.
```

Comment:
This may have been a mistake in the ticket. Could you please update to
```suggestion
 * @deprecated The `<IconWithFallback />` component has been deprecated in favor of the new `<AvatarNetwork />` or `<AvatarFavicon />` component from the component-library.
```